### PR TITLE
Qa 2105 test automation update 2.11.0 release

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Changed cohort creation flow to accommodate only one unsaved cohort
- Created a before suite step to setup runs before the specs execute

### CODE REVIEW

**Review by AmyCMP — COMMENTED** [review]
All the cohorts which have failed before are getting saved successfully but seen some additional 4 to 5 failures

**Comment by bpierce01** [comment]
> All the cohorts which have failed before are getting saved successfully but seen some additional 4 to 5 failures

Ran on prod-uat. Only 3 spec fails, all due to existing bugs with tickets made for them

File: holmes-py/specs/gdc_data_portal_v2/analysis_center/tools_text.spec [hunk 1]
```diff
@@ -23,7 +23,7 @@ tags: gdc-data-portal-v2, navigation, analysis-center
   |Age at Diagnosis |Apply                |
 * Wait for cohort bar case count loading spinner
 * Pause "2" seconds
-* Navigate to "Analysis" from "Header" "section"
+* Navigate to Analysis Center via URL
```
Comment (uc-pperumala) [inline holmes-py/specs/gdc_data_portal_v2/analysis_center/tools_text.spec:26]: I'm not too keen on changing this. This deviates us from using BDD format. Let me see if I can try to find an alternative.

**Review by AmyCMP — COMMENTED** [review]
I ran the changes in QA-UAT and found few issues, please look into this file attached for details...
[Gauge Test Results Dec 11th.pdf](https://github.com/NCI-GDC/gdc-frontend-framework/files/13639792/Gauge.Test.Results.Dec.11th.pdf)

File: holmes-py/specs/gdc_data_portal_v2/analysis_center/tools_text.spec (hunk 1 above)
Comment (bpierce01) [reply holmes-py/specs/gdc_data_portal_v2/analysis_center/tools_text.spec]: Put back and made additional waits

**Review by uc-pperumala — APPROVED** [review]
Change looks good! Thank you!

**Review by craigrbarnes — APPROVED** [review]

**Pull request merged**